great-idea/css/index.css

```diff
@@ -63,4 +65,277 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+.container {
+  width: 880px;
+  /* TB LR */
+  margin: 0 auto;
+}
+
+/* header */
+header {
+	display: flex;
+	justify-content: space-between;
+
+
+}
+
+
+header nav {
+  width: 650px;
+  margin-top: 35px;
+  display: flex;
+  justify-content: space-between;
+   
+}
+
+header nav a {
+  font-size: 14px;
+  color: gray;
+  text-decoration: none;
+}
+
+header nav a:hover {
+  text-decoration: underline;
+}
+
+header nav a:first-child {
+  margin-left: 0;
+}
+
+header nav a:last-child {
+  margin-right: 0;
+}
+
+header .logo {
+  display: flex;
+  margin-top: 25px;
+  
+}
+
+/* cta */
+
+.cta {
+  border: 2px solid purple;
+  display: flex;
+  flex-direction: row;
+}
+
+
+
+.cta .cta-text {
+  display: flex;
+  flex-wrap: wrap;
+  vertical-align: top;
+  font-size: 77px;
+  margin-top: 120px;
+  text-align: center;
+  padding: 0px 75px;
+  border: 2px solid red;
+  width: 500px;
+  justify-content: center;
+}
+
+.cta .cta-text button {
+  display: flex;
+  
+  background: white;
+  vertical-align: top;
+  margin-top: 6px;
+  padding: 8px 44px;
+  font-size: 17px;
+  border: 1px solid black;
+  cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+}
+
+.cta img {
+  display: flex;
+  margin-top: 69px;
+  border: 2px solid blue;
+  
+}
+
+.main-content {
+  border-top: 1px solid black;
+  margin-top: 76px;
+  padding-top: 39px;
+}
+
+.main-content .top-content {
+  padding: 0 10px;
+  display: flex;
+}
+
+.main-content .top-content .text-content {
+  width: 410px;
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  margin-right: 34px;
+  border: 2px solid yellow;
+  
+}
+
+.main-content .top-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.main-content .middle-img {
+  margin-top: 45px;
+}
+
+.main-content .bottom-content {
+  margin-top: 37px;
+  padding: 0 10px;
+  display: flex;
+  border: 2px solid pink;
+}
+
+
+.main-content .bottom-content .text-content {
+  width: 275px;
+  display: flex;
+  margin-right: 12px;
+}
+
+.main-content .bottom-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.contact {
+  margin-top: 57px;
+  border-top: 1px solid black;
+  padding: 39px 10px 0;
+}
+
+.contact p {
+  margin: 20px 0;
+}
+
+footer {
+  margin: 32px 0;
+  text-align: center;
+  font-size: 14px;
+}
+
+
+
+.services .services-header-img {
+   display: block;
+   margin: 30px 0;
+}
+
+.services .services-info {
+	padding: 30px;
+	border-bottom: 1px solid black;
+  margin-bottom: 30px;
+  
+
+}
+
+.container2 {
+  border: 2px solid blue;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+
+
+.services h1 {
+	font-size: 20px
+}
+
+.services .digital-info {
```

Comment:
This is a great place to reuse code. Refer to the solution and the way Josh uses the "card" class to only style the boxes once.